jetson/nav/rover.hpp

```diff
@@ -11,15 +11,17 @@
 #include "rover_msgs/Odometry.hpp"
 #include "rover_msgs/TargetList.hpp"
 #include "rover_msgs/Waypoint.hpp"
+#include "rover_msgs/AutonDriveControl.hpp"
 #include "rapidjson/document.h"
+#include "courseProgress.hpp"
+#include "environment.hpp"
 #include "pid.hpp"
 
-using namespace rover_msgs;
 using namespace std;
```

Comment:
can you get rid of this & just add the specific members of std that you need? like using std::vector? etc

Reply:
sure

Reply:
done